### PR TITLE
add some analytics

### PR DESCRIPTION
**Pull request by mfahler9**
add an analytics event to a purchase

### CODE REVIEW

File: apps/prenumerera/src/Analytics/Analytics.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
+if (typeof window !== "undefined") {
+    window.dataLayer = window.dataLayer || [];
+  }
```
Comment (jaakkoha-bytecraft) [inline apps/prenumerera/src/Analytics/Analytics.js:3]: `npx run prettier --write  apps/prenumerera/src/Analytics/Analytics.js`

File: apps/prenumerera/src/Page/Finish.purs [hunk 2]
```diff
@@ -5,26 +5,33 @@ import Prelude
...
   , method :: PaymentMethod
   }
 
 component :: Component Props
 component = do
   today <- Now.nowDate
-  React.component "Finish" \ { user, description, offer, method } -> do
+  React.component "Finish" \ { user, description, offer, method, package } -> React.do
     let summary = Summary.render today user description offer method
+    useEffect unit do
```
Comment (kaol) [inline apps/prenumerera/src/Page/Finish.purs:32]: `useEffectOnce` is better for this.

File: apps/prenumerera/src/Analytics/Analytics.purs [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+import Data.Int (toNumber)
+import Bottega.Models (PaymentMethod)
+import KSF.User (User)
+import Prenumerera.Package (Package, PackageOffer)
+
+foreign import _purchase :: EffectFn4 String String String Number Unit
+
+analyticsSendPurchase :: User -> Package -> PaymentMethod -> PackageOffer -> Effect Unit
+analyticsSendPurchase user package method offer = do
+    runEffectFn4 _purchase (toString user.cusno) package.id (show method) (toNumber offer.totalPrice/100.0)
```
Comment (jaakkoha-bytecraft) [inline apps/prenumerera/src/Analytics/Analytics.purs:17]: Here's also a missing newline from eof.

**Comment by jaakkoha-bytecraft** [comment]
Code itself looks fine, just some formatting nits.

**Review by kaol — APPROVED** [review]

**Pull request merged**